src/xdocs/config_coding.xml

```diff
@@ -3153,13 +3153,42 @@ public void foo(int i, String s) {}
         </p>
```

Comment:
But we do say that, `and (optionally) whether
          the package name matches the directory name for the source file.`